recipes/pytest-grpc/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-grpc" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-grpc-{{ version }}.tar.gz
+  sha256: 0bd2683ffd34199444d707c0ab01970b22e0afbba6cb1ddb6d578c85ebfe09bd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6.0
+  run:
+    - pytest >=3.6.0
+    - python >=3.6.0
+
+test:
+  imports:
+    - pytest_grpc
```

Comment:
Hello @dopplershift - thank you so much for having a look and also for the cool suggestion, haven't seen this pattern yet :) - so this will run the test suite and also check that the output contains the plugin - with the appropriate version?!

Reply:
No, in this case it starts up the pytest program and checks that its configuration finds the plugin.